docs/framework/docker/aspnetmvc.md

```diff
@@ -50,43 +45,39 @@ The finished application is located in the
 
 ## Prerequisites
 
-At a minimum, your development machine must be running
-[Windows 10 Anniversary Update](https://www.microsoft.com/en-us/software-download/windows10/)
-or [Windows Server 2016](https://www.microsoft.com/en-us/cloud-platform/windows-server). 
+The development machine must be running
+[Windows 10 Anniversary Update](https://www.microsoft.com/en-us/software-download/windows10/) (or higher)
+or [Windows Server 2016](https://www.microsoft.com/en-us/cloud-platform/windows-server) (or higher). 
 
-Before starting, you need to install [Docker for Windows](https://docs.docker.com/docker-for-windows/),
-version 1.12 Beta 26, or newer. Windows Container support is only available
-in the Beta channel at this time.
+[Docker for Windows](https://docs.docker.com/docker-for-windows/)
```

Comment:
I'm sure you're correct, but I don't understand why we need version specific info on this page. Shouldn't [Get started with Docker for Windows](https://docs.docker.com/docker-for-windows/) have all the info you need? I don't think we want to highlight the beta version, many folks will opt for the stable version.